en/sync-themes-plugin.md

```diff
@@ -54,7 +57,7 @@ Before you edit the Colors and Typography properties don't forget to specify the
 
 #### 2. Choosing brand and special colors
 
-In order to specify the two palettes at the top you are allowed to set only their base values: `Primary 500` and `Secondary 500` since the rest of the shades will be generated automatically for you. Most of the time you will use these palettes with your brand colors. As to `Error`, `Success`, `Warning`, and `Info` colors we strongly recommend to take in mind the users' perception of the application for displaying errors, success messages, warnings, and informative content. The default colors provided, are found to be meaningful and appropriate for the majority of these cases. Colors are easily adjustable by clicking on them and providing either a HEX/RGB value or picking the color manually.
```

Comment:
You can add a word about the surface color as well.

Reply:
I talk about it in the first sentence where I discuss the light/dark mode topic